core/src/main/resources/db/migration.sql

```diff
@@ -15,3 +15,10 @@ ALTER TABLE cna_event AUTO_INCREMENT=1;
 UPDATE info SET DB_SCHEMA_VERSION="1.2.0";
 ALTER TABLE mutation add UNIQUE KEY `UQ_MUTATION_EVENT_ID_GENETIC_PROFILE_ID_SAMPLE_ID` (`MUTATION_EVENT_ID`,`GENETIC_PROFILE_ID`,`SAMPLE_ID`);
 ALTER TABLE sample_profile add UNIQUE KEY `UQ_SAMPLE_ID_GENETIC_PROFILE_ID` (`SAMPLE_ID`,`GENETIC_PROFILE_ID`);
+
+##version: 1.2.1
+SET @s = (SELECT IF(    (SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS WHERE table_name = 'cancer_study' AND table_schema = DATABASE() AND column_name = 'STATUS') > 0,  "SELECT 1", " ALTER TABLE cancer_study ADD STATUS int(1) DEFAULT NULL"));
```

Comment:
Hi JJ, this code is the way I found for adding column `STATUS` to `cancer_study` only if it is not already there. I noticed that this column was missing in our DB, probably because it was introduced at a time where we did not yet have such migration scripts. So, basically the code runs `ALTER TABLE cancer_study ADD STATUS int(1) DEFAULT NULL` IF needed, and `SELECT 1` if not. 

Yes, this could also be set into 1.2.0....but then people who already migrated this will get the error that their DB is ahead of the code. Maybe better to keep as 1.2.1? I noticed I forgot to set 1.2.1 in cgds.sql (I set it to 1.2.0, while in pom.xml I correctly set it to 1.2.1 - should I fix it there?)